web/modules/custom/lex_custom/src/Plugin/Block/GTranslateBlock.php

```diff
@@ -0,0 +1,222 @@
+      EOT;
+
+      $default_language = $gtranslate_main_lang;
+      $block_content .= <<<EOT
+      <div id="google_translate_element2"></div>
+      <script>function googleTranslateElementInit2() {new google.translate.TranslateElement({pageLanguage: '$default_language', autoDisplay: false}, 'google_translate_element2');}</script>
+      <script src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit2"></script>
+      EOT;
+
+      switch($gtranslate_look) {
+        case 'overlay': {
+          $block_content .= '<button type="button" class="btn" data-toggle="modal" data-target="#translateModal">Translate</button>';
+          $block_content .= '<div class="modal fade" id="translateModal" tabindex="-1" role="dialog" aria-labelledby="translateModalTitle" aria-hidden="true">';
+          $block_content .= '<div class="modal-dialog" role="document">';
+          $block_content .= '<div class="modal-content">';
+          $block_content .= '<div class="modal-header">';
+          $block_content .= '<button id="close-icon" type="button" class="close" data-dismiss="modal" aria-label="Close">';
+          $block_content .= '<span aria-hidden="true">&times;</span>';
+          $block_content .= '</button>';
+          $block_content .= '</div>';
+          $block_content .= '<div class="modal-body">';
+          $block_content .= '<ul onchange="doGTranslate(this)" id="gtranslate_selector" class="notranslate" aria-label="Website Language Selector">';
+
+          foreach($languages as $lang => $lang_name) {
```

Comment:
Seemed like a good opportunity to move towards putting the markup in the twig and simply passing in `$languages`. Any particular reason you persisted this?